.github/workflows/htmlhint_linter.yml

```diff
@@ -0,0 +1,21 @@
+name: HTML Linter
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  lint:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
```

Comment:
per [this page](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions#create-an-example-workflow) these can be `@v3`. I don't know what the difference is, is there a reason to use `@v2`?

Reply:
It looks like some of the docs were only updated to v3 a few days ago, so that's probably the only reason we're using v2.